### DISCUSSION TITLE
Why is the MSSQL functionality not mentioned in the readme?

### DISCUSSION THREAD

**Discussion by senolfeldmann** [jeremyevans/sequel#2127] [General]
The title says it all. Shouldn't we use it? Is it not considered as stable? Why isn't it mentioned anywhere although it exists for ODBC and JDBC?

**Comment by jeremyevans** [comment]
Sequel's MSSQL support is mature, but the README mentions adapter support, not database support.  Sequel supports MSSQL via multiple adapters (tinytds, JDBC, ODBC).  The README doesn't mention support for access, db2, h2, hsqldb, or derby either.  I suppose we could additionally mention database support, as long as it fits in nicely with the surrounding text and doesn't appear redundant.

**Reply by senolfeldmann** [reply]
Thank you for your answer! I must have misread.
